region/wreckedship/main.json

```diff
@@ -831,17 +831,8 @@
           },
           "locks": [
             {
-              "unlock": [
-                {"or": [
-                  {"not": [
-                    "DefeatPhantoon"
-                  ]},
-                  {"and": [
-                    "DefeatPhantoon",
-                    "Kill All Enemies"
-                  ]}
-                ]}
-              ]
+              "lock": [ "DefeatPhantoon" ],
+              "unlock": [ "DefeatPhantoon", "Kill All Enemies" ]
```

Comment:
I think `DefeatPhantoon` is unnecessary here. It's implied that the door has been locked before being unlocked.